### PR TITLE
Server now has routing

### PR DESCRIPTION
**Pull request by manuscriptmastr**
@nybblr thoughts?

### CODE REVIEW

File: routes.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var Router = require('koa-router');
+var router = new Router();
```
Comment (nybblr) [inline routes.js]: I think you invoke this without the `new` operator?

Comment (manuscriptmastr) [reply routes.js]: Fixed.

File: templates/about.pug [hunk 2]
```diff
@@ -0,0 +1,10 @@
+doctype html
+html(lang= "en")
+  head
+    title about
+
+  body
+    h1 About The Developer
+
+    p.
+      Joshua Martin is a coffee connoisseur originally from Atlanta. He frequently haunts local coffee shops.
```
Comment (nybblr) [inline templates/about.pug]: How about a "latte connoisseur" instead? :)

**Comment by nybblr** [comment]
^^ just those two thoughts!

File: templates/about.pug (hunk 2 above)
Comment (manuscriptmastr) [reply templates/about.pug]: ;)

**Comment by nybblr** [comment]
:+1: 🚢 :shipit:

**Pull request merged**